WordPress/gutenberg
# Add the HTML element option to columns and column block

## Description

Adds the HTML element option to the columns and column blocks.
Closes #33343

## How has this been tested?




1. Add a columns block
2. Open the advanced section and change the HTML element
3. Save and view the front. Confirm that the selected HTML element is used.


## Types of changes




Enhancement

## Checklist:
